content/applications/websites/elearning.rst

```diff
@@ -0,0 +1,258 @@
+  submit reviews on your course;
+- :guilabel:`Forum`: add a dedicated forum to your course (only possible if **Forum** is installed);
+- :guilabel:`New Content Notification`: select an email template that is sent to your attendees when
+  you upload new content. Tick the green arrow to update the email template;
+- :guilabel:`Completion Notification`: select an email template that is sent to your attendees once
+  they reach the end of your course. Tick the green arrow to update the email template.
+
+.. _elearning/access-rights:
+
+Access rights
+*************
+
+- :guilabel:`Show course to`: define who can access your course and their content between
+  :guilabel:`Everyone`, :guilabel:`Signed In` or :guilabel:`Course Attendees`;
+- :guilabel:`Enroll Policy`: define how people enroll to your course. Select:
+
+   - :guilabel:`Open`: your course is available to anyone;
+   - :guilabel:`On Invitation`: if selected, set up the :guilabel:`Enroll message` explaining the
+     enroll process to subscribe to the course. This message appears on your website under the
+     course title;
+   - :guilabel:`On Payment`: the :guilabel:`Paid Courses` features must be enabled in your settings
+     to get this option. If you select :guilabel:`On Payment`, you must add a :guilabel:`Product`
+     for your course. Select an existing one or create a new one. It can be updated by clicking on
+     the green arrow that redirects you to the :doc:`../sales/sales` app to finalize the
+     configuration.
+
+.. _elearning/display:
+
+Display
+*******
+
+- :guilabel:`Training`: the course content appears as a training program, and the courses must be
+  taken in the proposed order.
+- :guilabel:`Documentation`: the content is available any order. If you chose this option, you can
+  select the :guilabel:`Featured Content` on the course homepage.
+
+.. _elearning/karma:
+
+Karma tab
+~~~~~~~~~
+
+This tab is about gamification to make e-learning fun and interactive.
+
+In the :guilabel:`Rewards` section, set up the rewards you want to grant your students when they
+:guilabel:`Review` or :guilabel:`Finish` a course.
+
+In the :guilabel:`Access Rights` section, define the karma needed to :guilabel:`Add Review`,
+:guilabel:`Add Comment`, or :guilabel:`Vote` for the course.
+
+.. note::
+   From your course, use the smart button :guilabel:`Contact attendees` to reach people who are
+   already part of the course.
+
+.. _elearning/create-content:
+
+Content
+-------
+
+Manage your content by going to :menuselection:`eLearning --> Courses --> Contents`. Click
+:guilabel:`New` to create content. Add your :guilabel:`Content Title`, and if you want
+:ref:`Tags <elearning/tags>`, then fill in the related information among the different tabs.
+
+.. image:: elearning/elearning-content-tab.png
+   :align: center
+   :alt: Create your content.
+
+Document tab
+~~~~~~~~~~~~
+
+- :guilabel:`Course`: select the course your content belongs to;
```

Comment:
:guilabel:`Course`: the course your content belongs to;

Reply:
I prefer to keep it as is, it's clearer according to me :)